integreat_cms/cms/forms/events/event_filter_form.py

```diff
@@ -60,7 +60,13 @@ class EventFilterForm(forms.Form):
         ),
         required=False,
     )
-
+    time_range_events = forms.TypedMultipleChoiceField(
```

Comment:
```suggestion
    time_range_events = forms.MultipleChoiceField(
```
No reason for a typed multi choice field, let's just use plain string values...